addons/hr/models/hr.py

```diff
@@ -369,3 +389,26 @@ def _update_employee_manager(self, manager_id):
                 ('parent_id', '=', department.manager_id.id)
             ])
         employees.write({'parent_id': manager_id})
+
+    @api.model_cr_context
+    def _init_column(self, column_name):
```

Comment:
Hello @switch87 ,

what's purpose of this method in department model ? department model does not have "barcode", "pin" columns.

Reply:
@pha-odoo